.travis.yml

```diff
@@ -57,11 +74,11 @@ matrix:
         # Fetch mbed-os: We use manual clone, with depth=1 and --single-branch to save time.
         - git clone --depth=1 --single-branch https://github.com/ARMmbed/mbed-os.git;
         # Install Mbed CLI and dependencies
-        - pip install --upgrade mbed-cli
+        - pip install --upgrade mbed-tools
```

Comment:
I think we need to keep building with Mbed CLI 1, as the tool is not deprecated yet. When that is done, we can remove it from the Travis job

Reply:
The other Mbed OS examples only test Mbed CLI 2 with Travis, so I tried to align this with these.
But it seems that they use Circle CI to test CLI 1, whereas this repo doesn't have that, so I'll add CLI 1 back.

@0xc0170 Maybe we could use Travis to test both CLI 1 & 2 for those examples in the future, as it reduces maintenance and complexity.

Reply:
Done

Reply:
Did you forget to put back `mbed-cli` here?

Reply:
Yes, I'll fix it

Reply:
Thinking about this, it's quite easy to add Circle CI so for consistency with the other examples, I would keep Circle CI to test CLI 1 and Travis for CLI 2.
cc @jamesbeyond

Reply:
Okay, I'll move CLI 1 to Circle CI

Reply:
@evedon I've added Circle CI to this PR, but don't have the permission to enable it.
The different between this example and others (e.g. blinky) is, we agreed (in December) to test all supported PSA targets, and the CI needs to handle parametrisation. I tried it: https://github.com/ARMmbed/mbed-os-example-psa/pull/20/commits/64e1184585dc6539e66c97da33c21edec08883c5 but I'm not sure if the syntax is correct, would need a Circle CI run to verify. Most other examples only test one target (K64F).